src/analyses/mallocWrapperAnalysis.ml

```diff
@@ -3,8 +3,9 @@
 open Prelude.Ana
 open Analyses
 open GobConfig
+open ThreadIdDomain
 
-module Spec : Analyses.MCPSpec =
+module Spec () : Analyses.MCPSpec =
```

Comment:
IIRC, I think it was needed to avoid accessing some configuration things before the config is parsed.

Reply:
Ah, I guess it's the `let n` depending on the option for construction of `Lattice.Chain`. I suppose the most consistent thing to do would be to change it to `val n: unit -> int` in `Printable.ChainParams`. We already have it like that for `name`, `bot`, `top` and `arbitrary`. It was probably never done for `Chain`, because it's used very little and for constant things.

Reply:
Done!